test/unit/phys/parameters/output_parameters/output_parameters_test.cpp

```diff
@@ -19,7 +19,7 @@ TEST(OutputParametersTest, DefaultValues) {
   dca::phys::params::OutputParameters pars;
 
   EXPECT_EQ("./", pars.get_directory());
-  EXPECT_EQ("HDF5", pars.get_output_format());
```

Comment:
see above, output format is still should be selectable.

Reply:
reverted in e23561ac5229177383403d00e2abdde3bdca7cc0